functions/inputs/from.go

```diff
@@ -3,529 +3,75 @@ package inputs
 import (
 	"fmt"
 	"github.com/influxdata/flux"
-	"github.com/influxdata/flux/ast"
-	"github.com/influxdata/flux/execute"
-	"github.com/influxdata/flux/functions"
-	"github.com/influxdata/flux/functions/transformations"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/semantic"
-	"github.com/influxdata/flux/values"
-	"github.com/pkg/errors"
 )
 
-const FromKind = "from"
+const MockFromKind = "mockFrom"
```

Comment:
Ultimately I think it makes sense to have `from` live in the `platform` repo since the `flux` repo should be source-agnostic.  However, the `influxdb` repo (1.x version of the product) shares some code with `platform`, but has a separate implementation of `from`.  So whatever change is made should work for both 2.0 and 1.x.  There are longer-term plans to merge `influxdb` and `platform` into one repo I think?  I think we need some broader discussion on this.

I think it makes sense to have a `mockFrom` that lets us test `flux` in isolation.  But I don't think that it belongs in this directory with product code.  We could make it so that an `init` method in the `querytest` package will register `mockFrom`, so maybe it should live in `flux/querytest`.

Tests that need `mockFrom` to be available when compiling Flux can then just import `querytest` in addition to `flux/builtin` to get everything they need.

Reply:
👍

Reply:
By the way, you can make it so that `mockFrom` does not need to have a range pushed into it.  That may make it easier to use.

There is at least one controller test that makes use of `from` needing a `range` pushed into it to introduce a planner failure.  I guess in that case you'll need to define a `mockRangeNeedingFrom` operation or something similar.